src/test/java/com/github/wasiqb/coteafs/datasource/XmlTest.java

```diff
@@ -0,0 +1,64 @@
+package com.github.wasiqb.coteafs.datasource;
+
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import com.github.wasiqb.coteafs.datasource.data.XmlData;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @author Faisal Khatri
+ * @since Aug 31, 2020
+ */
+public class XmlTest {
+
+    /**
+     * @author Faisal Khatri
+     * @since Aug 31, 2020
+     * @return testdata
+     */
+    @Test
```

Comment:
Same change here as mentioned in previous comments.

Reply:
Done